user/drawterm/template.py

```diff
@@ -0,0 +1,33 @@
+pkgname = "drawterm"
+_commit = "f11139d4c918802a87730bc14d094670ee4ce572"
+pkgver = "0_git20240703"
+pkgrel = 0
+build_style = "makefile"
+make_env = {"CONF": "linux"}
+make_use_env = True
+hostmakedepends = [
+    "pkgconf",
+    "wayland-progs",
+]
+makedepends = [
+    "libxkbcommon-devel",
+    "linux-headers",
+    "pipewire-devel",
+    "wayland-devel",
```

Comment:
building the wayland gui means it's likely tied to just wayland compositors and quite possibly only wlroots which is sketch

Reply:
nothing, seems fine

Reply:
for linux, they explicitly only build GUI=wl. i guess you could do a second build with GUI=x11